bin/plugin/lib/milestone.js

```diff
@@ -1,7 +1,12 @@
-/** @typedef {import('@octokit/rest')} GitHub */
-/** @typedef {import('@octokit/rest').IssuesListForRepoResponseItem} IssuesListForRepoResponseItem */
-/** @typedef {import('@octokit/rest').IssuesListMilestonesForRepoResponseItem} OktokitIssuesListMilestonesForRepoResponseItem */
-/** @typedef {import('@octokit/rest').ReposListReleasesResponse} ReposListReleasesResponse */
+/** @typedef {import('@octokit/rest').Octokit} GitHub */
+/** @typedef {import('@octokit/types').Endpoints} Endpoints */
+/** @typedef {Endpoints["GET /repos/{owner}/{repo}/issues"]["response"]} listIssuesResponse  */
+/** @typedef {Endpoints["GET /repos/{owner}/{repo}/milestones"]["response"]} listMilestonesResponse  */
```

Comment:
JSDoc flags these as syntax errors (maybe because of the curly braces around `owner` and `repo`? or because of the whitespace? I haven't really been able to track down the exact issue).

Reply:
Maybe try 2 spaces around the type, I think I ran into a similar problem myself once and the spaces helped if I recall correctly

```suggestion
/** @typedef { Endpoints["GET /repos/{owner}/{repo}/issues"]["response"] } listIssuesResponse  */
/** @typedef { Endpoints["GET /repos/{owner}/{repo}/milestones"]["response"] } listMilestonesResponse  */
```

Reply:
Hmm, seems like it's still failing :confused: But thanks anyway!